### DISCUSSION TITLE
夸克盘挂接错误：  require login [{0}]

### DISCUSSION THREAD

**Discussion by Ciddmao** [AlistGo/alist#2376] [Q&A]
前几天突然Alist页面空白，重新安装了Alist 3。在挂接夸克盘时，出现出错：require login [{0}]。
Cookie的值抓取应该没错，之前也都是可以用的。重装好开始输入错了，但现在删除后，再挂接，一直提示这个错误。

**Comment by libtantang** [comment]
搭车同问

**Comment by HelloTrouble** [comment]
搭车同问
